Homework-1/task_2.py

```diff
@@ -0,0 +1,8 @@
+
+# 2. პროგრამამ უნდა შეგაყვანინოთ სახელი და დაბადების წელი. ეკრანზე უნდა გამოვიდეს, გამარჯობა, „თქვენს მიერ შეყვანილი სახელი“, თქვენ ხართ „თქვენი წლოვანება“ წლის.
+
+name = input("Enter your name ")
+yearOfBirth = int(input("Enter your year of birth "))
```

Comment:
პითონში სტანდარტია `snake_case`. 
```suggestion
year_of_birth = int(input("Enter your year of birth "))
```